src/pagesComponents/Auth/LeftPanel/LeftPanel.jsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import { useNavigate } from 'react-router-dom';
+import PropTypes from 'prop-types';
+import Tabs from './Tabs';
+import Title from './Title';
+import Button from '../Button';
+import { Form, FormField, SubmitButton } from '../../../components';
+import { Container, TitleWrapper } from './styles';
+
+export default function LeftSide({ title }) {
+  const navigate = useNavigate();
+  const url = window.location.search.substring(1);
+  return (
+    <Container>
+      {title === 'Verify email' || url === 'verify-email' ? (
```

Comment:
The passing of the value `title` is unnecessary. You can remove it entirely and all its calls and nothing would break.